test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+public class UtilsTest {
+    
+    @Test
+    public void testNullItems() {
+        assertTrue(Utils.isAnyNull( (Object) null ));
+    }
+    
+    @Test
+    public void testNonNullItems() {
+        assertFalse(Utils.isAnyNull( new Object() ));
+    }
+
+    @Test
+    public void testNonUnique() {
+        List<Integer> list = Arrays.asList(1,1,1,1,1);
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.